src/mapbox/mapbox.js

```diff
@@ -37,6 +37,7 @@ const propTypes = {
   gl: PropTypes.object, /** External WebGLContext to use */
 
   mapboxApiAccessToken: PropTypes.string, /** Mapbox API access token for Mapbox tiles/styles. */
+  mapboxApiUrl: PropTypes.string,
```

Comment:
Add this to the `StaticMap` and `InteractiveMap` components documentation as well?

Reply:
@Pessimistress thanks for 👀; updated 👍